src/js/api/fetchMoviesMain.js

```diff
@@ -0,0 +1,25 @@
+const fetchMoviesMain = async () => {
+    try {
+    const data = await fetch("https://api.themoviedb.org/3/trending/all/day?api_key=4f9b3bc6cd1b3d6e0d830ad9a5ccfefd");
```

Comment:
pytanie co będziesz chciał zrobić przy następnych funkcjach (będziemy mieli api_key w wielu miejscach) ale narazie chodzi o to żeby to działało :)